reports/_model.qmd

```diff
@@ -12,6 +12,317 @@ training_data_pred <- training_data %>%
   )
 ```
 
+
+## Overall Ratio Distribution
+```{r, fig.width=10, fig.height=6}
+shorten_number <- function(num) {
+  if (num >= 1000) {
+    return(paste(format(round(num / 1000), nsmall = 0)))
+  }
+  return(as.character(num))
+}
+
+data_to_plot <- model_performance_quantile_test %>%
+  filter(
+    geography_type == "triad_code",
+    by_class == FALSE,
+    triad_code == run_triad_code,
+    num_quantile == "10"
+  ) %>%
+  mutate(
+    lower_bound_short = sapply(lower_bound, shorten_number),
+    upper_bound_short = sapply(upper_bound, shorten_number),
+    custom_label = paste(quantile,
+      "\n[$",
+      lower_bound_short,
+      "K",
+      "-\n",
+      "$",
+      upper_bound_short,
+      "K",
+      "]",
+      sep = ""
+    )
+  )
+
+ggplot(data_to_plot, aes(x = quantile, y = median_ratio)) +
+  geom_rect(aes(xmin = -Inf, xmax = Inf, ymin = 0.95, ymax = 1.05),
+    fill = "#D3F2C2", alpha = 0.1
+  ) +
```

Comment:
**nitpick:** For the sake of readability, let's put arguments on their own line in cases like this, i.e.:

```suggestion
  geom_rect(
    aes(xmin = -Inf, xmax = Inf, ymin = 0.95, ymax = 1.05),
    fill = "#D3F2C2",
    alpha = 0.1
  ) +
```